events/kossprint.md

```diff
@@ -0,0 +1,22 @@
+# KOSSprint
+KOSS internal code sprint.
+
+## What is KOSSprint?
+- It's an internal event held to efficiently collaborate on large projects or accelerate slowed-down work, where KOSS members sit together for 2-3 hours and work on KOSS projects, slides, documentation, etc.
+- It boosts productivity and encourages discussion and collaboration among KOSS members.
+
+## When to conduct a KOSSprint?
+- Sprints can be done whenever a lot of coding work (or any other work) is pending.
+eg:
+  - For planning a workshop, all can work together to make slides and poster.
+  - During KWoC, all can work together to update the frontend, backend, mailing, etc.
+- Suggested days:
+  - Weekends
+- Suggested times:
+  - 10pm - 1am  
+
+## Where can KOSSprint be conducted?
+- The place should have a good internet connection, enough plug points, and no time restrictions (as sprints are usually held around midnight).
+- Some suggested venues:
+  - Maggu Room
+  - 
```

Comment:
Do you know any other venue? If not, this can be removed for now and updated in the future.

Reply:
library? :skull:

Reply:
corridors of main building first floor

Reply:
> library? 💀

Ig we can add this.

Reply:
> corridors of main building first floor

Are sockets available?

Reply:
hm, no sockets in corridors

okay, library added